app/slimming/slimming.js

```diff
@@ -1,11 +1,30 @@
 'use strict';
 app.controller('GOSlimCtrl', function($scope, $location, $q,
-  hardCodedDataService, presetsService, termService, basketService,
+  hardCodedDataService, presetsService, $document, termService, basketService,
```

Comment:
Hard coded data is used to retrieve the taxa information. Presets and taxonomy service should be used for that.
Take a look at taxonFilter.js to see how to retrieve the presets and then to annotationList.js to see how to complete info with the taxonomy service.